articles/cdn/cdn-app-dev-node.md

```diff
@@ -103,7 +103,7 @@ With *app.js* open in our editor, let's get the basic structure of our program w
    
     ``` javascript
     var credentials = new msRestAzure.ApplicationTokenCredentials(clientId, tenantId, clientSecret);
-    var cdnClient = new cdnManagementClient(credentials, subscriptionId);
+    var cdnClient = new cdnManagementClient.CdnManagementClient(credentials, subscriptionId);
```

Comment:
> new cdnManagementClient.CdnManagementClient(...)

This is confusing. I would recommend updating the require statement as `const { CdnManagementClient } = require("@azure/arm-cdn");`  Then this becomes `new CdnManagementClient (...)`

Previously, the client was the default export from the cdn package, so the user could just "new" it up.
Now, we have multiple exports from the package.

Reply:
changed. Thanks